lib/ex_schedule.ex

```diff
@@ -132,8 +132,12 @@ defmodule ExSchedule do
       defp normalize_namespace(ns) when is_bitstring(ns), do: String.to_atom(ns)
       defp normalize_namespace(ns), do: ns
 
-      defp child_spec(schedule) do
-        worker(ExSchedule.ScheduledTask, [schedule], id: schedule.id)
+      defp start(args) do
+        [head | tail] = args
+
+        children = [
+          {ExSchedule.ScheduledTask, head}
```

Comment:
This will always return a list with a **single** child spec, but we need to return a child spec spec for each scheduled task.

We could change `child_spec/1` to return the new map child specs with:
 
```elixir
defp child_spec(schedule) do
  %{id: schedule.id, start: {ExSchedule.ScheduledTask, :start_link, [schedule]}}
end
```